dev-system/project.clj

```diff
@@ -135,10 +135,16 @@
             ;; Alias to test2junit for consistency with lein-test-out
             "test-out"
             ["test2junit"]
-            ;; Installs the Elasticsearch Marvel plugin locally.
-            ;; Visit http://localhost:9210/_plugin/marvel/sense/index.html
-            "install-marvel"
-            ["shell" "cmr" "install" "local" "marvel"]
+            ;; Install spatial plugin locally
+            "install-spatial-plugin"
+            ["shell" "cmr" "install" "local" "spatial_plugin"]
+            ;; Install with local elasticsearch
+            "install"
```

Comment:
Shouldn't this be `"install!"` and with line 146 and 147 removed?

Reply:
Hmm i was thinking we wanted the plugin to install with `lein install`. Think i should change so it only gets installed with `lein install!`?

Reply:
nm. I think you are right.